### ISSUE TITLE
Add audio track when using Pipe input

### ISSUE THREAD

**Issue by Namaneo** [rosenbjerg/FFMpegCore#198]
Hello!

I've been playing with conversions using pipe input and output. I have this piece of code:
```csharp
var inputPipe = new RawVideoPipeSource(Frames.GetConsumingEnumerable()) { FrameRate = 60 };
return FFMpegArguments
    .FromPipeInput(inputPipe)
    .OutputToFile("test.mp4", true, options => options
        .WithVideoCodec("h264")
        .ForceFormat("mp4"))
    .ProcessAsynchronously();
```
How can I add an audio track to this conversion, keeping it "real-time", without having to save and rework a physical video file? I believe that there is something to do with the `.AddPipeInput()` method, but I couldn't find how to use it properly.

As for some context, I'm experimenting with an implementation of Libretro in .NET (using native bindings to the pre-built binaries of Retroarch cores). I basically receive one frame and one audio sample each emulation iteration, and I want ultimately to stream all of this to a remote front-end (not sure which for now, but this might be a web UI using something like WebRTC): `.OutputToFile()` will one day (?) be replace with a `.OuputToPipe()`.

Thanks!

**Comment by Namaneo** [comment]
Got a working implementation of a `RawAudioPipeSource`. 🙂 
Once I finish to cleanup the code, I'll open a PR!

**Comment by rosenbjerg** [comment]
Closed by #204
